docs/docs/tutorials/multirobot.mdx

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 4
+title: Multi Robot Simulation
+---
+
+This page provides a collection of tutorials that describe how to simulate, launch controllers, and teleoperate multiple BlueROVs. Prior to starting these tutorials, you should have completed the [Running Blue in Simulation](/tutorials/simulation),the [Integrating Custom Controllers](/tutorials/control), and Keyboard teleoperation section in[Teleoperation](/tutorials/teleop) tutorials.
+
+## Multi Robot Simulation and Teleoperation
+
+This tutorial describes how to launch multiple BlueROVs in Gazebo with non-conflicting communication channels, independent controllers, and perform keyboard based teleoperation. It also demonstrates spawning static and dynamic obstacles in the environment.
+
+<div align="center">
+   <img src="../../static/img/multi-robot-sim/multi-rob.png" alt="My Image" width="300"/>
+</div>
+
+### Dependencies
+
+The following ROS 2 dependencies are required for this tutorial:
+
+- Gazebo Harmonic or newer
+- [ardusub_driver](https://github.com/Robotic-Decision-Making-Lab/ardusub_driver)
+- [auv_controllers](https://github.com/Robotic-Decision-Making-Lab/auv_controllers)
+- [teleop_twist_keyboard](https://github.com/ros2/teleop_twist_keyboard)
+- [mobile_to_maritime](https://github.com/Robotic-Decision-Making-Lab/mobile_to_maritime)
+
+These dependencies will be met by default if you have installed Blue with
+Docker as described in the [installation instructions](/installation).
+
+### Tutorial steps
+
+1. Launch the demo Dependencies for `rob_1` and `rob_3` instances in simulation by running the following commands:
+
+   ```bash
+   ros2 launch blue_demos rob_1_bluerov2_heavy_demo.launch.yaml use_sim:=true
+   ```
+
+   ```bash
+   ros2 launch blue_demos rob_3_bluerov2_heavy_demo.launch.yaml use_sim:=true
+   ```
+
+2. Once Gazebo, ArduSub, and MAVROS have fully loaded, open a new terminal and launch the demo control framework for `rob_1` and `rob_3` instances:
```

Comment:
2. Once Gazebo, ArduSub, and MAVROS have fully loaded, open two new terminals. In the first terminal, launch the control framework for `rob_1`

```bash
ros2 launch blue_demos multi_robot_1_controllers.launch.py use_sim:=true
```

and in the second terminal, launch the control stack for `rob_2`:

```bash
ros2 launch blue_demos multi_robot_2_controllers.launch.py use_sim:=true